### PR TITLE
"url 404"

### PR DESCRIPTION
**Pull request by wyyxd2017**
fix a 404 url

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,7 +198,7 @@ and/or other countries."
 [gcal]: https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com
 [prow]: https://go.k8s.io/bot-commands
 [new_issue]: https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/new
-[slack_info]: https://github.com/kubernetes/community/blob/master/communication.md#social-media
+[slack_info]: https://github.com/kubernetes/community/blob/master/events/2019/05-contributor-summit/communications.md#social-media
```
Comment (ashish-amarnath) [inline README.md]: I think this link can be removed and replace it with `[slack]` instead.

Comment (wyyxd2017) [reply README.md]: I agree with you,this link is a Description of doker slack.

Comment (wyyxd2017) [reply README.md]: ok,I agree with you.

**Pull request merged**